.github/actions/setup-zeebe/action.yml

```diff
@@ -33,10 +37,10 @@ runs:
       with:
         distribution: 'temurin'
         java-version: ${{ inputs.java-version }}
-    - if: ${{ inputs.java == 'true' }}
+    - if: ${{ inputs.java == 'true' && inputs.maven == 'true' }}
       uses: stCarolas/setup-maven@v4.4
       with:
-        maven-version: ${{ inputs.maven-version }}
+        maven-version: '3.8.5'
```

Comment:
:x: The input is still there, see line 23.

Reply:
Good catch, my bad :see_no_evil: